src/Microsoft.ML/CSharpApi.cs

```diff
@@ -8028,28 +8028,28 @@ public BinNormalizer(params string[] inputColumns)
                 }
             }
             
-            public BinNormalizer(params ValueTuple<string, string>[] inputOutputColumns)
```

Comment:
>ValueTuple [](start = 40, length = 10)

Woo hoo, we can remove dependency on that package! I'll create issue after we merge this one.